packages/mobile/src/pincode/PincodeSet.tsx

```diff
@@ -84,6 +88,7 @@ export class PincodeSet extends React.Component<Props, State> {
     return this.state.pin1 === pin
   }
 
+  // Function for PIN to be changed
```

Comment:
This is not true, this function is also used during onboarding. Maybe we can just delete it?

Reply:
deleted the comment